deploy/crds/wildfly_v1alpha1_wildflyserver_crd.yaml

```diff
@@ -12,6 +12,8 @@ spec:
     shortNames:
     - wfly
   scope: ""
+  subresources:
```

Comment:
since #88 I fixed code generation to make sure we don't edit this file manually.

Was this added by `make codegen`?

Reply:
@jmesnil yes, that was added by `make codegen` but I needed to remove several other changes that running `codegen` on my machine changes the paths (like instead of `./pkg/apis/wildfly/v1alpha1.StandaloneConfigMapSpec` it changes to `github.com/wildfly/wildfly-operator/pkg/apis/wildfly/v1alpha1.StandaloneConfigMapSpec`)

I have not much idea why my `codegen` generates different results than the one on your machine.

Does your run of `make codegen` generates what is in the `wildfly_v1alpha1_wildflyserver_crd.yaml`? Why the `subresources` was removed from what was generated before #88?

Reply:
I'm not sure why to be honest...

`status` is now generated as a top level field: https://github.com/wildfly/wildfly-operator/pull/88/files#diff-aa625220cf655743036e4c98524baaf2R23

I'm using `operator-sdk version: v0.10.0, commit: ff80b17737a6a0aade663e4827e8af3ab5a21170` to generate the files. Which version are you using?